angular_hit.m

```diff
@@ -40,45 +40,47 @@
     % Solve the systems Az=b to check for intersection
     Amin = [xmin, -ray_direction(1); ymin, -ray_direction(2)];
```

Comment:
The math isn't entirely intuitive for me here. What do Amin and Amax represent? If its simple, you can just explain it here. If its complex, I'd add a few more comments for Youhan and I to reference later.

Reply:
We are solving the system [rcos(xmin)-circle_center(1) rsin(xmin)-circle_center(2)]' = [u0 u1]' + [v0 v1]'t and same for xmax ymax as args to trig functions. The var array is [r t]; so A[r t] = b.

Reply:
Sorry, I don't think that's so clear. I'm trying to make dinner and type at the same time. Will be back around 8 pm.